lib/clientStub.js

```diff
@@ -166,6 +167,12 @@ var DgraphClientStub = (function () {
                     value: "true",
                 });
             }
+            if (((_a = req === null || req === void 0 ? void 0 : req.hash) === null || _a === void 0 ? void 0 : _a.length) > 0) {
```

Comment:
I wonder why there are so many checks for "hash". Is it because it is a string?

Reply:
Yeah so we are using optional chaining for checking if its higher than 0 the length, since we strict checking enabled which doesn't allow for non-boolean expression in if clause.